src/NuGet.Core/NuGet.Protocol/Resources/PackageUpdateResource.cs

```diff
@@ -221,7 +221,7 @@ private static string GetSourceDisplayName(string source)
             return "'" + source + "'";
         }
 
-        private static IEnumerable<string> GetPackagesToPush(string packagePath)
+        public static IEnumerable<string> ResolvePackageFromPath(string packagePath)
```

Comment:
Move these wildcard methods to a new static helper class. The Sign command shouldn't have a dependency on Push.

LocalFolderUtility might be a good place to add them.